### PR TITLE
docs: clarify subscription config

### PR DESCRIPTION
**Pull request by Meschreiber**
Quick clarification on the `path` configuration because it caused customer confusion,
In the long run, this page should be combined with https://www.apollographql.com/docs/graphos/operations/subscriptions , refactored, and include a reference for all configurations.

### CODE REVIEW

**Review by kendallshearman — COMMENTED** [review]
Looks great! Thanks so much for all you do 🎉

File: docs/source/executing-operations/subscription-support.mdx [hunk 1]
```diff
@@ -99,13 +99,20 @@ subscription:
...
 ```
 
 This example enables subscriptions in **passthrough mode**, which uses long-lived WebSocket connections.
-Note: If your subgraph implementation (e.g. [DGS](https://netflix.github.io/dgs/)) can close idle connection, set `heartbest_interval` to keep connection alive.
+
+<Note>
+
+- Each `path` must be set as an absolute path. For example, given `http://localhost:8080/foo/bar/graphql/ws`, set the path configuration as `path: "/foo/bar/graphql/ws"`.
+- Subgraph path configurations override the path configuration for `all` subgraphs.
+- If your subgraph implementation (e.g. [DGS](https://netflix.github.io/dgs/)) can close idle connections, set `heartbest_interval` to keep the connection alive.
```
Comment (shorgi) [inline docs/source/executing-operations/subscription-support.mdx:113]:
```suggestion
- If your subgraph implementation (e.g. [DGS](https://netflix.github.io/dgs/)) can close idle connections, set `heartbeat_interval` to keep the connection alive.
```

File: docs/source/executing-operations/subscription-support.mdx [hunk 2]
```diff
@@ -99,13 +99,20 @@ subscription:
         path: /subscriptions # The absolute URL path to use for subgraph subscription endpoints (Default: /ws)
       subgraphs: # Overrides subscription settings for individual subgraphs
         reviews: # Overrides settings for the 'reviews' subgraph
-          path: /ws # Absolute path that overrides '/subscriptions' defined above
+          path: /ws # Absolute path that overrides the '/subscriptions' path defined above
```
Comment (shorgi) [inline docs/source/executing-operations/subscription-support.mdx:102]:
```suggestion
          path: /ws # Absolute path that overrides the preceding '/subscriptions' path for 'all'
```

**Review by shorgi — APPROVED** [review]

**Review by Geal — APPROVED** [review]

**Comment by abernix** [comment]
@Meschreiber I'm going to see if merging in `main` fixes the failing `test_updated` check, which [looked](https://app.circleci.com/pipelines/github/apollographql/router/22197/workflows/b10cee66-9581-4557-8e40-bc1a75064571/jobs/153016) legit to me — not because of your fault, but because this is merging directly to `main` it could leave `main` in a broken state.

**Comment by Meschreiber** [comment]
Thank you @abernix !

**Pull request merged**